### ISSUE TITLE
Nautilus/Nemo draws an ugly frame border on thumbnails bigger than 128px

### ISSUE THREAD

**Issue by mttbernardini** [mttbernardini/appimage-thumbnailer#3]
In order to avoid the frame we need to cap the max size to `127px` (or actually `52px` to make the icon have the same size of native mime-icons).

This approach, however, has the drawback of affecting the image quality. I don't know if there's any other alternative, since the "framing" seems to be hardcoded:

https://github.com/linuxmint/nemo/blob/7fe83632eca5801a55992dc08c52767b6d8685cb/libnemo-private/nemo-file.c#L4817-L4820

Therefore, for now I will mark this issue as "wontfix".

**Comment by mttbernardini** [comment]
Confirmed that this issue applies to original GNOME Files (aka Nautilus) as well:

https://github.com/GNOME/nautilus/blob/master/src/nautilus-file.c#L5236-L5254

**Comment by probonopd** [comment]
Yes, this is very annoying but not unique to this thumbnailer. I guess this behavior would have to be fixed in GNOME.

**Comment by JustTryingToGetSomeWorkDone** [comment]
I think this needs to be reconsidered. I do not see a frame around any big icons in Nemo in Mint 22.3. Please reconsider allowing the GNOME based thumbnailer to specify any icon size.
